Signal/src/ViewControllers/ConversationView/ConversationInputTextView.m

```diff
@@ -12,7 +12,6 @@ @interface ConversationInputTextView () <UITextViewDelegate>
 
 @property (nonatomic) UILabel *placeholderView;
 @property (nonatomic) NSArray<NSLayoutConstraint *> *placeholderConstraints;
-@property (nonatomic) BOOL isEditing;
```

Comment:
Separate fix: we want to show the input placeholder whenever the text input is empty, even if user is editing.